tests/system/migrations/test_0068_transfer_history_to_history_collection.py

```diff
@@ -23,7 +23,11 @@ def write_many(
                 {
                     "type": "create",
                     "fqid": f"{collection}/{id_}",
-                    "fields": {"id": id_, namekey: f"{collection}{id_}{suffix}"},
+                    "fields": {
+                        "id": id_,
+                        namekey: f"{collection}{id_}{suffix}",
+                        **({"meeting_id": 1} if collection != "user" else {}),
```

Comment:
I think this should work also and is more explicit.
```suggestion
                        "meeting_id": 1 if collection != "user" else None),
```